core/src/test/scala/akka/kafka/internal/ConsumerTest.scala

```diff
@@ -120,7 +120,7 @@ class ConsumerTest(_system: ActorSystem)
 
       probe
         .request(1)
-        .expectNoMsg()
+        .expectNoMessage(200.millis)
```

Comment:
I wonder if we should provide a version of this method with a default timeout here after all (though independent from and smaller than the default expect timeout, see https://github.com/akka/akka/pull/23381#discussion_r140767414)